index.d.ts

```diff
@@ -31,7 +31,7 @@ export interface IDataTableProps<T> {
     rows: T[],
     field: string,
     sortDirection: 'asc' | 'desc'
-  ) => T[];
+) => T[];
```

Comment:
nit pick - just move this back to 2 spaces

Reply:
Thank you for reviewing! I have just corrected!